utils/tasks.js

```diff
@@ -84,6 +84,7 @@ const transformQuery = (dev = false, status = "", size, page, assignee = "", tit
     dev: transformedDev,
     assignee: transformedAssignee,
     title: transformedTitle,
+    assigneeRole: assigneeRole,
```

Comment:
If there is no operation being done on assigneeRole is it necessary to pass it to this function?

Reply:
removed it as it is not needed